### DISCUSSION TITLE
Error after Update

### DISCUSSION THREAD

**Discussion by HomaEEE** [filamentphp/filament#8087] [Help]
Have some error on production server with Cloudlfare

Livewire - 3.0.1
Filament 3.0.34

Snapshot missing on Livewire component with id: 9S0HcdDZEGikS66Isj3d

can't load and display any modal and slideover window

https://github.com/filamentphp/filament/assets/1429343/84efab71-102e-49b3-9663-17e1441f935a

Cache and views are cleaned

**Comment by suleymanozev** [comment]
@danharrin please convert to issue

**Reply by danharrin** [reply]
No, you can submit it as it needs a reproduction repository

**Reply by suleymanozev** [reply]
unfortunately this error occurs in production. Local environment works fine

**Reply by suleymanozev** [reply]
At least that's how it is with me

**Reply by danharrin** [reply]
if you cant even replicate it, how is the Filament team supposed to be able to fix it? this is exactly why we don't want issues without reproduction repsitories
